client/src/modules/list/model/selectors.js

```diff
@@ -1,24 +1,29 @@
 import _pick from 'lodash/pick';
 import _filter from 'lodash/filter';
 import _keyBy from 'lodash/keyBy';
+import { createSelector } from 'reselect';
 
 export const getList = (state, listId) =>
   _pick(state.lists.data, listId)[listId];
-export const getItems = (state, listId) => {
-  const currentList = getList(state, listId);
 
-  if (currentList) {
-    const { items } = currentList;
-    return (
-      items &&
-      items.sort((a, b) =>
-        new Date(a.createdAt).getTime() > new Date(b.createdAt).getTime()
-          ? -1
-          : 1
-      )
-    );
+export const getItems = createSelector(
+  [getList],
+  list => {
+    if (list) {
+      const { items } = list;
+
+      return (
+        items &&
```

Comment:
coudl `items` be set to `[]` by default? then we wouldn't have to check if that exists